dev/conductor/core/test/codesign_integration_test.dart

```diff
@@ -2,6 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// This test clones the framework and downloads pre-built binaries; it sometimes
+// times out with the default 5 minutes: https://github.com/flutter/flutter/issues/100937
+@Timeout(Duration(minutes: 10))
```

Comment:
It seems like the bot got throttled from GitHub and that's why this flakes.

It'd be nice if we could make conductor checkout against GoB, and then set the upstream back to GitHub.

Reply:
that wouldn't work pre-submit

Reply:
Wouldn't it work in this case? Or is the assumption this test might have a breaking change where you need the changes in presubmit? If so, we could copy the current framework checkout to a tmp folder.